src/core/styles/common/defaultTheme.ts

```diff
@@ -13,6 +13,31 @@ enum FontWeight {
 }
 
 export const defaultAppTheme: AppTheme = {
+  borders: {
```

Comment:
opted to group them by color to closer match how the colors are grouped, but open to changing this